README.md

```diff
@@ -1 +1,26 @@
-# enumerable
\ No newline at end of file
+# enumerable
+Replicate enumerables any?, all? and filter.
+
+# Built with
+- Ruby
+
```

Comment:
- [ ] Documenting your code is one of the core skills you need to learn as a junior developer. And The microverse projects are a really great place to test your skills and get feedback on them. I recommend that you document all your projects properly. Please add the necessary directives to run and use the project.